dxr/query.py

```diff
@@ -78,23 +110,32 @@ def _line_query_results(self, filters, results, path_highlighters):
     def _file_query_results(self, results, path_highlighters):
         """Return an iterable of results of a FILE-domain query."""
         for file in results:
-            yield (icon(file['path'][0]),
-                   highlight(file['path'][0],
+            yield ('folder' if file['is_folder'] else icon(file['path'][0]),
+                   highlight_path(file['path'][0],
                              chain.from_iterable(
                                  h(file) for h in path_highlighters)),
                    [],
                    file.get('is_binary', False))
 
+    def _instantiate_filters(self, terms):
```

Comment:
The methods on Query (and most of the rest of the project) are tending toward nouns (single_term(), direct_result(), results()). Let's either stick with that for consistency (and inline this into `__init__` to avoid a collision on "_filters") or name it _make_filters() (consistent with _make_es_query()).